resources/views/statuslabels/edit.blade.php

```diff
@@ -71,11 +71,14 @@
 
 @section('moar_scripts')
     <!-- bootstrap color picker -->
+    <?php 
```

Comment:
We try not to set PHP tags and variables directly in the blade. I think we can just do this in the function below.